### DISCUSSION TITLE
Disable fs access for multiwebviews

### DISCUSSION THREAD

**Discussion by sebhernandezr** [tauri-apps/tauri#10375] [Q&A]
I have an app that uses the beta multiwebview features. Given a webview with an external URL, when the website is trying to access the fs (either for saving or for uploading something), then I want to completely disable the access. I don't want to see the native fs at all.

I have played around with the [tauri_plugin_fs](https://docs.rs/tauri-plugin-fs/latest/tauri_plugin_fs/struct.Scope.html) to find a workaround, like setting `allow_directory` to empty, but no luck.

Is there a way to accomplish this?

**Comment by FabianLars** [comment]
Are we really talking about the fs plugin apis (and other plugins)? Because then you'd just don't give those webviews permissions in the capabilities, or use `deny-<api>` instead of `allow-<api>` if the webview inherits permissions from another capability.

If we're talking about the webview's own file system access like via `input type="file"` then you/we don't have any control over that.
